rav1e_wasm/README.md

```diff
@@ -0,0 +1,34 @@
+# rav1e_wasm
+
+- [🚴 Usage](#-usage)
```

Comment:
Do we require to have this table here,
Seems to be a stray change for a small readme, I would avoid it

Reply:
It's there because I would like to establish consistent styling for all the docs. It matches the style of #2365.

But actually we could move this to the main README, since it is so short

Reply:
Sure, 
Once that PR is merged, we can have it